<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DriverInfo</class>
 <widget class="QMainWindow" name="DriverInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#MainWindow { font: 16pt &quot;Aganè S&quot;; }
#MainWindow { background-color : #9CAFD1  }
QPushButton {color : white;
background-color : #3562B0;
border-radius: 10px;}
QPushButton{ font: 81 14pt &quot;AXIS Extra Bold&quot;;}
#pushButton_Login_2{ font: 81 18pt &quot;AXIS Extra Bold&quot;;
border-radius: 25px; }
QGroupBox{ background-color : #EEEEEE;}
QLabel{  font: 16pt &quot;Aganè S&quot;; }
QStackedWidget{ background: qlineargradient(x1:0, y1:0, x2:1, y2:1,  stop:0 #202020, stop: 0.4 #303030, stop:1 #353535)}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string/>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0" rowspan="2">
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="label">
           <property name="font">
            <font>
             <family>Aganè S</family>
             <pointsize>16</pointsize>
             <weight>50</weight>
             <italic>false</italic>
             <bold>false</bold>
             <underline>true</underline>
            </font>
           </property>
           <property name="text">
            <string>Name</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="name_label"/>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_driver_name_Change">
           <property name="text">
            <string>Change</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="font">
            <font>
             <family>Aganè S</family>
             <pointsize>16</pointsize>
             <weight>50</weight>
             <italic>false</italic>
             <bold>false</bold>
             <underline>true</underline>
            </font>
           </property>
           <property name="text">
            <string>Email</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="email_label"/>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_driver_email_Change">
           <property name="text">
            <string>Change</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_5">
           <property name="font">
            <font>
             <family>Aganè S</family>
             <pointsize>16</pointsize>
             <weight>50</weight>
             <italic>false</italic>
             <bold>false</bold>
             <underline>true</underline>
            </font>
           </property>
           <property name="text">
            <string>Points</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="points_label">
           <property name="font">
            <font>
             <family>Aganè S</family>
             <pointsize>16</pointsize>
             <weight>50</weight>
             <italic>false</italic>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLineEdit" name="lineEdit_pointUpdate">
             <property name="text">
              <string>Update points</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QPushButton" name="add_points">
               <property name="text">
                <string>Add</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="subtract_points">
               <property name="text">
                <string>Subtract</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLabel" name="label_6">
           <property name="font">
            <font>
             <family>Aganè S</family>
             <pointsize>16</pointsize>
             <weight>50</weight>
             <italic>false</italic>
             <bold>false</bold>
             <underline>true</underline>
            </font>
           </property>
           <property name="text">
            <string>License Number</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="ln_label"/>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_driver_ln_Change">
           <property name="text">
            <string>Change</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_7">
           <property name="font">
            <font>
             <family>Aganè S</family>
             <pointsize>16</pointsize>
             <weight>50</weight>
             <italic>false</italic>
             <bold>false</bold>
             <underline>true</underline>
            </font>
           </property>
           <property name="text">
            <string>License Plate Number</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lp_label"/>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_driver_lp_Change">
           <property name="text">
            <string>Change</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="1">
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="font">
            <font>
             <family>Aganè S</family>
             <pointsize>16</pointsize>
             <weight>50</weight>
             <italic>false</italic>
             <bold>false</bold>
             <underline>true</underline>
            </font>
           </property>
           <property name="text">
            <string>Recent Orders</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTableWidget" name="tableWidget_3"/>
         </item>
        </layout>
       </item>
       <item row="1" column="1">
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="font">
            <font>
             <family>Aganè S</family>
             <pointsize>16</pointsize>
             <weight>50</weight>
             <italic>false</italic>
             <bold>false</bold>
             <underline>true</underline>
            </font>
           </property>
           <property name="text">
            <string>Address</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="address_label"/>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_driver_address_Add">
           <property name="text">
            <string>Change</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
